12024 Fix Filings UI - Display "Not Entered" for missing data on address and prop/part components

lewischenstudio opened the pull request:
*Issue #:* [/bcgov/entity#12024](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/12024)

*Description of changes:*
- Change the color for "(Not Entered)" to $gray9 (#212529) under Proprietor/Partner list


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

severinbeauvais commented on package-lock.json:
```diff
@@ -1,12 +1,12 @@
 {
   "name": "business-filings-ui",
-  "version": "5.1.0",
+  "version": "5.1.1",
```
If you want this deployed through to Test today or tomorrow (ie, Rls 13.2a) then please rebase to the release-5.0.4 branch.

(Your current changes are built upon my date changes, which I don't want to take to Test right away.)

lewischenstudio replied:
Okay. Let me rebase to that branch first.

severinbeauvais commented on src/components/Dashboard/ProprietorPartnersListSm.vue:
```diff
@@ -154,6 +154,10 @@ $avatar-width: 2.75rem;
 
+.complete-filing.not-entered {
+  color: $gray9;
+}
```
Looks good.

severinbeauvais commented on tests/unit/NameRequestInfo.spec.ts:
```diff
@@ -10,8 +10,8 @@ const vuetify = new Vuetify({})
-localVue.filter("VMask", data => data) // mock filter
-localVue.filter("capitalize", data => data) // mock filter
+localVue.filter('VMask', data => data) // mock filter
+localVue.filter('capitalize', data => data) // mock filter
```
Ah, oops, thanks for the lint cleanup :)

lewischenstudio replied:
It's done automatically. 😄

severinbeauvais commented:
1. if possible, don't chec in the new unit test file

2. change this PR's target to release-5.0.4 branch

severinbeauvais commented on tests/unit/NameRequestInfo.spec.ts:
```diff
@@ -0,0 +1,184 @@
+    expect(vals.at(3).text()).toBe('APPROVED')
+    expect(vals.at(4).text()).toBe('Not Required')
+
+    expect(firstCol.find('#status .nr-status-icon.mdi-check').exists()).toBe(false)
```
No, you shouldn't have to do this (but your codebase doesn't have the latest NameRequestInfo component).

If possible, remove this file from your PR. It might actually be easier to close this PR and start a new one?

lewischenstudio replied:
> 1. if possible, don't check in the new unit test file
> 2. change this PR's target to release-5.0.4 branch

I've changed the base to release-5.0.4 branch and removed NameRequestInfo.spec.ts file. I hope it's okay, or I will close this one and start a new one.

severinbeauvais replied:
Looks good! I'll merge it when the checks finish.

lewischenstudio commented:
> 1. if possible, don't check in the new unit test file
> 2. change this PR's target to release-5.0.4 branch

I've changed the base to release-5.0.4 branch and removed NameRequestInfo.spec.ts file. I hope it's okay, or I will close this one and start a new one.
